src/intro/mod.rs

```diff
@@ -0,0 +1,92 @@
+extern crate ggez;
+use ggez::*;
```

Comment:
Wildcard includes are generally discouraged, for two reasons:

* It is less clear where a certain symbol comes from when reading the code.
* A future version of the library may contain more symbols and they may collide with something already imported from somewhere else.

Reply:
You are right, fixed in 5f8ce91bdb059c9cf4bd0b28de3c4febcc81bb47